ftplugin/dirvish.vim

```diff
@@ -19,6 +19,8 @@ nnoremap <buffer><silent>   g?    :help dirvish-mappings<CR>
 execute 'nnoremap '.s:nowait.'<buffer> x :Shdo  {}<Left><Left><Left>'
 execute 'xnoremap '.s:nowait.'<buffer> x :Shdo  {}<Left><Left><Left>'
 
+execute 'nnoremap '.s:nowait.'<buffer> gh :call dirvish#toggle_dotfiles()<CR>'
```

Comment:
Up to now the plugin has intentionally avoided adding any public functions except for `dirvish#open`. Don't want to start proliferating functions for this purpose.